### DISCUSSION TITLE
Similar functionality to next/after

### DISCUSSION THREAD

**Discussion by paulm17** [QwikDev/qwik-evolution#23] [Proposals For Qwik]
### What is it about?

> after() is a new experimental API that solves this problem by allowing you to schedule work to be processed after the response has finished streaming, enabling secondary tasks to run without blocking the primary response.

#### Problems you are trying to solve:
* Offloading server/database functionality within a request

I have a use case where I need to server static content, however I also want to offload a visitor count (in-house analytics) to a separate (golang) server to increment the count in a db.  

What vercel is proposing with next/after is exactly what I am looking for.  

Btw, my use case is adtech.  Which is why with qwik, I want to keep things as static as possible for maximum speed.

#### Goals you are trying to achieve:
* https://nextjs.org/blog/next-15-rc#executing-code-after-a-response-with-nextafter-experimental

#### Any other context or information you want to share:

https://www.youtube.com/watch?v=ECnlX00YcPI

TLDW:
- Bun beats golang for static content
- Bun 50% slower than golang with db interactivity

#### What do you propose?

Same functionality as proposed by vercel: 
https://nextjs.org/blog/next-15-rc#executing-code-after-a-response-with-nextafter-experimental

**Comment by wmertens** [comment]
Since this is specific to serverless code, I think this should go in serverless adapters.

For regular servers, you can easily start running a task and then respond, all you need to do is not await the Promise (but keep track of it somewhere)
